tests/ffi/agent-flood/test.sh

```diff
@@ -106,8 +106,50 @@ run_test_containers
 for ((i=1;i<=NUMBER_OF_NODES;i++)); do
     if [ "$(podman exec qm systemctl is-active bluechi-tester-"${i}")" != "active" ]; then
         info_message "test() bluechi-tester-${i} is not active"
+        info_message "test()1 bluechi-tester-${i} logs..."
```

Comment:
Consider removing all those debug messages
In case test failed need to debug it later on